_includes/v21.1/orchestration/start-cockroachdb-operator-secure.md

```diff
@@ -17,86 +13,58 @@ The Operator is currently supported for GKE only.
 
 1. Apply the Operator manifest:
 
+    {{site.data.alerts.callout_info}}
+    By default, the Operator is configured to install in the `default` namespace. To use the Operator in a custom namespace, download the [Operator manifest](https://github.com/cockroachdb/cockroach-operator/blob/master/manifests/operator.yaml) and edit all instances of `namespace: default` to specify your custom namespace. Then apply this version of the manifest to the cluster with `kubectl apply -f {local-file-path}` instead of using the command below.
```

Comment:
Maybe a callout that the operator can only install CRDB into the same namespace that it is installed in?

Reply:
👍